packages/xrpc-server/tests/subscriptions.test.ts

```diff
@@ -99,12 +95,10 @@ describe('Subscriptions', () => {
     const countdown = Number(params.countdown ?? 0)
     for (let i = countdown; i >= 0; i--) {
       yield {
-        $type:
-          i % 2 === 0 ? 'io.example.stream2#even' : 'io.example.stream2#odd',
+        $type: i % 2 === 0 ? '#even' : '#odd',
```

Comment:
I think in a perfect world you can yield `{ $type: 'io.example.stream2#even' }`, and `#even` get's placed as the discriminator in the header.  I saw the compaction of the `$type` as being sort of an implementation detail of the wire protocol, and the application having the ability to deal with lex unions just as it normally would.  Interested to hear your thinking on it!

Reply:
yup yup you're right. I fixed this up 👍